sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -1076,7 +1130,8 @@
                       },
                       "tableView": true,
                       "validate": {
-                        "required": true
+                        "required": true,
+                        "custom": "const currentDateFormatted = moment().format('YYYY-MM-DD');\nvalid = input >= data.applicationOfferingUnformattedStartDate && input <= data.applicationOfferingUnformattedEndDate && input <= currentDateFormatted ? true : 'Date should be between study start date, end date and current date.';"
```

Comment:
Would recommend to get business opinion on error message. `Date should be between study start date, end date and current date.`